sample-app/src/main/java/com/joinforage/android/example/ui/pos/data/PosPaymentRequest.kt

```diff
@@ -10,7 +10,9 @@ data class PosPaymentRequest(
     @Json(name = "payment_method") val paymentMethodRef: String,
     val description: String,
     @Json(name = "pos_terminal") val posTerminal: PosTerminal,
-    val metadata: Map<String, String>
+    val metadata: Map<String, String>,
+    @Json(name = "transaction_type") val transactionType: String? = null,
+    @Json(name = "cash_back_amount") val cashBackAmount: Double? = null
```

Comment:
damn these `@Json` tags with the `@JsonClass` seem really convenient!

Reply:
right? so nice to send a snake case key name in json but just get to use it as camel case in kotlin

Reply:
I feel like we were chatting about wanting to do this in our repos at some point to keep the responses strongly typed but I think at the time it felt like it could be a "big deal".....but this....this feels not so bad...almost easy??